ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -40,6 +40,8 @@ import SwiftUI
     @objc @Published public var footnoteTrailingAccessoryView: UIView?
     @objc @Published public var trailingView: UIView?
     @objc @Published public var accessoryType: MSFListAccessoryType = .none
+    @objc @Published public var cellBackgroundColor: UIColor?
```

Comment:
This is already the cell state. Can we name the custom properties like the tokens class?

```
class MSFListCellTokens: MSFTokensBase, ObservableObject {
    @Published public var backgroundColor: UIColor!
    ...
    @Published public var highlightedBackgroundColor: UIColor!
```